resources/postgres-setup-scripts/README.md

```diff
@@ -0,0 +1,29 @@
+Below you can find instructions on a recently automated way of running the initial data setup scripts using a single command. 
+
+1.To install PostgreSQL on WSL (ie. Ubuntu):
+Open your WSL terminal (ie. Ubuntu).
+Update your Ubuntu packages: sudo apt update
+Once the packages have updated, install PostgreSQL (and the -contrib package which has some helpful utilities) with: sudo apt install postgresql postgresql-contrib
+Confirm installation and get the version number: psql --version
+
+2.Download the script package from repo into your WSL: (link is coming)
```

Comment:
Here you want to additionally specify that the script directory will correspond to the scheme being set up. 

Your V1 is currently set as a single script which is fine, but this now needs to move the current scripts into an SFI22 directory because they are specifically for SFI22 setup only. This will be the same for the readme in each as some tables will change or have additional queries to make work

We can go over SFI23 next, so you may as well also create the SFI23 directory and we can set up the scripts for that next.